README.md

```diff
@@ -149,9 +149,9 @@ curl http://localhost:6800/listjobs.json
 ```
 ```json
 {
-  "finished":[],
+  "finished":[{"id":"fbjffo6tz5al5qjft0zrv0t7b0vzccxx","project":"example","spider":"static","state":"finished", "start_time":"2012-09-12 10:10:00.000000", "end_time":"2012-09-12 10:20:00.000000"}], // End time will only be available for the k8s launcher (when the job is finished and not cancelled), for docker it will always be null
```

Comment:
@wvengen do you think this is sufficient? Or should we also mention something in the **API** > **listjobs.json** section in the readme?

Reply:
Ouch, this is ugly. This output shows what you would get when running this. You don't see this comment.

I would expect this either in a section describing current features and limitations, or perhaps in our case, in the list of endpoints supported. There you can indicate what job status does not support.

Reply:
Added a section for limitation to the listjobs sections in the readme.